trace/benchmark_test.go

```diff
@@ -94,10 +153,12 @@ func BenchmarkSpanID_DotString(b *testing.B) {
 func traceBenchmark(b *testing.B, fn func(*testing.B)) {
 	b.Run("AlwaysSample", func(b *testing.B) {
 		b.ReportAllocs()
+		b.ResetTimer()
```

Comment:
the order of calling ResetTimer is different here than the below. could we ensure that we call ResetTimer as the very last thing before fn?